src/main/java/pro/trevor/tankgame/Main.java

```diff
@@ -4,21 +4,23 @@
 import org.json.JSONObject;
 import pro.trevor.tankgame.rule.impl.util.ApiRegistry;
 import pro.trevor.tankgame.rule.impl.IApi;
-import pro.trevor.tankgame.rule.impl.version3.Api;
+import pro.trevor.tankgame.rule.impl.version3.ApiV3;
+import pro.trevor.tankgame.rule.impl.version4.ApiV4;
 
 import java.io.File;
 import java.nio.file.Files;
 
 public class Main {
 
     public static void main(String[] args) {
-        ApiRegistry.putApi("3", Api.class);
+        ApiRegistry.putApi("3", ApiV3.class);
+        ApiRegistry.putApi("4", ApiV4.class);
 
         if (args.length == 1 && (args[0].equals("--debug") || args[0].equals("-d"))) {
             // Demo version 3 rules with game logs
             File initialFile = new File("example/initial.json");
             File movesFile = new File("example/moves.json");
-            IApi api = new Api();
+            IApi api = new ApiV3();
```

Comment:
Would it be cleaner to name every new API `ApiV_` or would it be cleaner to use fully qualified names here? Ex: `pro.trevor.tankgame.rule.impl.version3.Api` or `pro.trevor.tankgame.rule.impl.version4.Api`

Reply:
I think fully qualified names are a bad idea here since it makes it take up so much space. I initially had that but decided against it.